packages/cli-repl/src/smoke-tests.ts

```diff
@@ -397,7 +397,7 @@ async function runSmokeTest({
   // eslint-disable-next-line
   const { spawn } = require('child_process') as typeof import('child_process');
   const proc = spawn(executable, [...args], {
-    stdio: ['pipe', 'pipe', includeStderr ? 'pipe' : 'inherit'],
+    stdio: 'pipe',
```

Comment:
Changing to `pipe` indiscriminately so that we can then validate there's no warnings in stderr.